csaf-lib/src/csaf/csaf2_1/validation.rs

```diff
@@ -1,28 +1,32 @@
 use super::product_helper::*;
 use super::schema::CommonSecurityAdvisoryFramework;
 use crate::csaf::helpers::find_duplicates;
-use crate::csaf::validation::{Test, Validatable, ValidationPreset};
+use crate::csaf::validation::{test_6_01_35_contradicting_remediations, Test, Validatable, ValidationPreset};
```

Comment:
Why do we need 6.1.35 explicit, all other not?

Reply:
Until this PR, tests have been copy-pastet for 2.0 and 2.1. This is the first test residing in a common location, using our own Traits. Therefore, it has to be imported here.

Reply:
@tschmidtb51 It just came to my mind that this poses another interesting question.
Per spec, all test from 6.1.34 onward are CSAF 2.1 only. However, they can still make sense for CSAF 2.0 documents. Max recursion (6.1.34) and contradicting remediations (6.1.35) are good examples for this.
Should it be possible to run these tests for 2.0 documents, at least manually?
Or shall I move the test code to the 2.1 package in case we do not want to allow this?
@oxisto Any further thoughts on this?

Reply:
Technically, since they are only CSAF 2.1 tests, they should only be run on CSAF 2.1 documents, since otherwise we would declare a CSAF 2.0 document invalid because it fails a CSAF 2.1 test.

Reply:
@oxisto Regarding presets, absolutely agreed! How about manual runs on specific documents?